packages/vapor/scss/components/modal.scss

```diff
@@ -258,3 +258,9 @@ $icon-size: (4em / 3);
     align-items: center;
     margin-bottom: 21px;
 }
+
+.ReactModalPortal {
+    &.force-z-index {
+        z-index: 100000;
```

Comment:
I am not sure about this solution, it will make the dropdowns contained by the modal appear underneath it